src/lib/browse/layers/points/CycleParking.svelte

```diff
@@ -39,33 +40,20 @@
 <VectorTileSource
   url={`pmtiles://${publicResourceBaseUrl()}/v1/${name}.pmtiles`}
 >
-  <CircleLayer
+  <SymbolLayer
     {...layerId(name)}
     sourceLayer={name}
-    paint={{
-      "circle-color": color,
-      "circle-radius": [
-        "interpolate",
-        ["linear"],
-        ["zoom"],
-        1,
-        2,
-        8,
-        3,
-        13,
-        10,
-      ],
-    }}
     layout={{
+      "icon-image": "cycle_parking",
+      "icon-size": 1.0,
+      "icon-allow-overlap": true,
```

Comment:
I'm in favor of this for both. It looks kind of messy at low zoom, but having icons appear as you zoom in is really disorienting to me